compiler/rustc_feature/src/unstable.rs

```diff
@@ -502,6 +502,8 @@ declare_features! (
     (incomplete, fn_delegation, "1.76.0", Some(118212)),
     /// Allows impls for the Freeze trait.
     (internal, freeze_impls, "1.78.0", Some(121675)),
+    /// Frontmatter `---` blocks for use by external tools.
+    (unstable, frontmatter, "CURRENT_RUSTC_VERSION", Some(136889)),
```

Comment:
Discussion (meta, not necessarily for this PR): hm, frontmatter is going to be interesting when it comes to cooking it sufficiently for stabilization. Like `rustfmt`, `clippy`, `rust-analyzer`, `cargo` might all at least need to learn how to account for (or properly ignore) frontmatters in source files, since it's a *language* syntax and not just sth tools can be expected to naively strip away prior to passing a modified source file to rustc.

Reply:
Registered this as an unresolved question on the tracking issue, resolving as out of scope.